server/services/crmDisplayService.test.ts

```diff
@@ -116,49 +80,7 @@ describe('CRM Display Service', () => {
           { href: '/crm5/1234567/standard-properties', text: 'Standard Properties', active: true },
           { href: '/crm5/1234567/general-information', text: 'General Information', active: false },
           { href: '/crm5/1234567/firm-details', text: 'Firm Details', active: false },
-          { href: '/crm5/1234567/clients-details', text: "Client's Details", active: false },
-          { href: '/crm5/1234567/court-of-appeal-funding', text: 'Court of Appeal Funding', active: false },
-          { href: '/crm5/1234567/details-of-work-completed', text: 'Details of Work Completed', active: false },
-          { href: '/crm5/1234567/costs', text: 'Costs', active: false },
-          { text: 'Case History', href: '/crm5/1234567/case-history', active: false },
-          { text: "Solicitor's Certification", href: '/crm5/1234567/solicitors-certification', active: false },
-          { href: '/crm5/1234567/office-use-only', text: 'Office Use Only', active: false },
-          { href: '/crm5/1234567/evidence', text: 'Evidence', active: false },
-          { href: '/crm5/1234567/further-information', text: 'Further Information', active: false },
-          { href: '/crm5/1234567/summary', text: 'Summary', active: false },
-        ],
-        label: 'Side navigation',
-      })
-    })
-
-    it('should return crm navigation for conditions met', () => {
```

Comment:
This test case needs to be reinstated as it was previously, as it tests the showWhen condition.